packages/app/components/claim/claim-limit-explanation.tsx

```diff
@@ -0,0 +1,36 @@
+import { BottomSheetModalProvider } from "@gorhom/bottom-sheet";
+
+import { Text } from "@showtime-xyz/universal.text";
+import { View } from "@showtime-xyz/universal.view";
+
+import { useMyInfo } from "app/hooks/api-hooks";
+
+export const ClaimLimitExplanationModal = () => {
+  const { data: userProfile } = useMyInfo();
+
+  const hasVerifiedPhoneNumber =
+    userProfile?.data?.profile.has_verified_phone_number;
+  const dailyClaimLimit = userProfile?.data?.daily_claim_limit ?? 5;
+  return (
+    <BottomSheetModalProvider>
+      <View tw="mb-6 justify-center px-4 pt-4">
+        <Text tw="text-sm text-black dark:text-white">
+          {`- Your claims are precious — use them wisely!`}
+        </Text>
+        <View tw="h-4" />
+        <Text tw="text-sm text-black dark:text-white">
+          {hasVerifiedPhoneNumber ? (
```

Comment:
this copy maybe still need to change, Rize has some comments, waiting for Alex to reply.